js/app.js

```diff
@@ -1,9 +1,110 @@
 // wait for DOM to load before running JS
-$(document).ready( function() {
 
-  // check to make sure JS is loaded
-  console.log('JS is loaded!');
+$(document).ready( function() {
+  let searchInput = 'Linkin Park';
+  let type = 'track';
+  let artistHTML = function(artistName) {
+    $('h2').html(`
+      ${artistName}
+      <br>
```

Comment:
Ray,

Code looks good. I really like your use of comments but going forward be mindful of leaving in commented out code and console logs when you're submitting your homework. Now that we're moving into week 3 we're going to be making sure you guys are using best practice when writing code.

Keep up the good work!

Matt